### ISSUE TITLE
The  constituency parse result  is incorrect when using srparser/englishSR.ser.gz.

### ISSUE THREAD

**Issue by d0ngw** [stanfordnlp/CoreNLP#1250]
Input text: In 1972, American fell in love with two giant pandas named Hsing-Hsing and Ling-ling. 
CoreNLP Version: 4.3.0, 4.4.0 

The result from lexparser/englishPCFG.ser.gz:
Constituency parse:
```
(ROOT
  (S
    (PP (IN In)
      (NP (CD 1972)))
    (SYM ，)
    (NP (NNP American))
    (VP (VBD fell)
      (PP (IN in)
        (NP
          (NP (NN love))
          (PP (IN with)
            (NP
              (NP (CD two) (JJ giant) (NNS pandas))
              (VP (VBN named))))
          (NML
            (NML (NNP Hsing) (HYPH -) (NNP Hsing))
            (CC and)
            (NML (NNP Ling) (HYPH -) (NNP ling))))))
    (. .)))
```

The result from srparser/englishSR.ser.gz:
Constituency parse:
```
(ROOT
  (S
    (PP (IN In)
      (NP
        (NML (CD 1972) (SYM ，))
        (NNP American)))
    (VP (VBD fell)
      (PP (IN in)
        (NP (NN love)))
      (PP (IN with)
        (NP
          (NP
            (NP (CD two) (JJ giant) (NNS pandas))
            (VP (VBN named)
              (S
                (NP (NNP Hsing) (HYPH -) (NNP Hsing)))))
          (CC and)
          (NP (NNP Ling) (HYPH -) (NNP ling)))))
    (. .)))
```

The srparser is fast, but the result seems to be incorrect.

**Comment by AngledLuffa** [comment]
Both parses are total garbage fires, so ...

The SR parser is noticeably more accurate according to the F1 score on the PTB test set.  I recommend using it even though occasionally there are absolutely going to be parses which are better for one model or the other.

You'll probably also get better results in general if you replace `，` with `,` as there are literally zero examples of wide comma in any of the training data that went into either model.

**Comment by d0ngw** [comment]
Thanks for the suggestion, we will continue to use SR parser.
